src/components/Search.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+class Search extends React.Component {
+    constructor() {
+        super();
+
+        this.state= {
+            query: ''
+        }
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(event) {
+        this.setState({
+            query: event.target.value
+        })
+    }
+
+    handleSubmit(event) {
+        event.preventDefault();
+        this.props.receiveQuery(this.state.query);
+        document.querySelector(".search__input").value = "";
```

Comment:
It's really bad practice to use query selector in React. A better approach would be to set query value in state to be an empty string, then you can add a value attribute on the input and use `this.state.query` as the value. That way when we update it in state, render will automatically output what we want